packages/itinerary-body/src/TransitLegBody/index.js

```diff
@@ -39,6 +39,37 @@ DefaultTransitLegSummary.propTypes = {
   stopsExpanded: PropTypes.bool.isRequired
 };
 
+function DefaultRouteDescription({ leg }) {
+  const { headsign, routeLongName, routeShortName } = leg;
+  return (
+    <Styled.LegDescriptionForTransit>
+      {routeShortName && (
+        <div>
```

Comment:
Does the `<div>` element accomplish anything that `<ItineraryBodyClasses.LegDescriptionRouteShortName>` doesn't already do?

Reply:
Yes, it has some styles applied from the `LegDescriptionForTransit` styling rules. Then, the `LegDescriptionRouteShortName` contains a margin property that is able to be applied since this div in question has a display type of `table-cell` which disables the effect of margin settings.

Reply:
A table??